tests/gpflow/models/test_gpr_posterior.py

```diff
@@ -21,9 +20,9 @@ def make_models(regression_data):
 
 def _get_data_for_tests():
     """Helper function to create testing data"""
-    X = np.random.randn(100, input_dim)
-    Y = np.random.randn(100, output_dim)
-    X_new = np.random.randn(100, input_dim)
+    X = np.random.randn(5, 2)
+    Y = np.random.randn(output_dim)
+    X_new = np.random.randn(10, 5, 2)
```

Comment:
Perhaps worth testing for even more leading dims? i.e. `X_new = np.random.rand(3,10,5,2)`

Reply:
yes... I had to rework my solution to handle this....so good call. It now works and I've updated the tests to reflect this use case

Reply:
Great!